specifyweb/frontend/js_src/lib/components/Interactions/InteractionDialog.tsx

```diff
@@ -106,15 +119,15 @@ export function InteractionDialog({
             headers: { Accept: 'application/json' },
             body: {
               recordSetId: recordSet?.id ?? undefined,
-              loanNumbers: recordSet === undefined ? items : undefined,
+              loanNumbers: recordSet === undefined ? catalogNumbers : undefined,
             },
           }
-        ).then(({ data }) =>
+        ).then(({ data }) => {
```

Comment:
don't need `{` and `;` and `}` in here.
are those being added accidentally, or is it your preference to always add them?

I usually don't add them unless necessary as excessive curly braces make code refactoring harder (it's easy to miss one curly brace somewhere or add an extra one and suddenly you get a syntax error and don't know where)